schema.graphql

```diff
@@ -26,13 +26,15 @@ type HistoryElement @entity {
   method: String!
   address: String! @index
   networkFee: String!
-  success: Boolean!
+  execution: ExecutionResult!
```

Comment:
@bgodlin it's a significant change that may break mobile applications. Let's discuss it on the mobile meeting.